build-tools/packages/version-tools/.eslintrc.cjs

```diff
@@ -17,6 +17,11 @@ module.exports = {
 		"@typescript-eslint/strict-boolean-expressions": "off",
 
 		// This package is exclusively used in a Node.js context
-		"import/no-nodejs-modules": "off",
+		"import-x/no-nodejs-modules": "off",
+
+		// Rules that moved from eslint-plugin-import to eslint-plugin-import-x
+		"import/no-default-export": "off",
+		"import/no-deprecated": "off",
+		"import/no-named-as-default-member": "off",
```

Comment:
If the comment is accurate, these would need to read `import-x/` instead of `import/`, no? I also wonder if they're actually necessary if they weren't there before.

Reply:
These should all be removed.